src/utils/componentCategories.ts

```diff
@@ -116,6 +116,7 @@ const categories = {
   navigation: {
     title: "Components/Navigation",
     components: [
+      "AlphabetFilter",
```

Comment:
In the Visual Design Library (the Figma Main file), `AlphabetFilter` is listed under `Form Elements`, but because this component is similar to `Pagination`, we talked about moving it to `Navigation` (which I prefer).  That said, leave this as is.  I will update the Visual Design Library and I will also start to include a component's `Category` in the spec doc.